celery/worker/request.py

```diff
@@ -517,11 +517,7 @@ def on_failure(self, exc_info, send_failed_event=True, return_ok=False):
 
         # These are special cases where the process would not have had time
```

Comment:
should we change the comments as well?

Reply:
I think it wouldn't hurt to leave the comments intact as they have to do with the `WorkerLostError` and `return_ok == False` special cases